modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -240,10 +240,14 @@ export default class I3SConverter {
 
     try {
       const preloadOptions = await this._fetchPreloadOptions();
+      let tilesetUrl = inputUrl;
```

Comment:
The code is a bit difficult to follow. I think it is partly because the existing helper function is called `_fetchPreloadOptions` but this update makes it clear that the main thing it returns a URL, and beyond that only the headers of the options are used.

`_preloadURLAndHeaders()` ?

Reply:
The response of prefetch:
![image](https://github.com/visgl/loaders.gl/assets/18549629/521b48cf-623f-4f94-a7e4-e08b820e5ddd)
There are other properties that we don't need